arangod/Transaction/Manager.cpp

```diff
@@ -45,19 +45,20 @@
 #include <velocypack/Iterator.h>
 #include <velocypack/velocypack-aliases.h>
 
-
 namespace arangodb {
 namespace transaction {
 
+const size_t Manager::maxTransactionSize;  // 128 MiB
```

Comment:
This `maxTransactionSize` limit is actually still enforced.
I understood from today's discussion with Ingo that we only want to have the idle timeout, but no other restrictions yet. Did I misunderstand? @ifcologne ?

Reply:
I think the max size is still good, I thought its about the overall max time which we want to avoid

Reply:
We want to enforce `maxTransactionSize` (128MB) and `idleTTL` (10Sec).

We do not want to enforce the limit on the total lifespan of a transaction.

Reply:
Got it. I misunderstood it then. All good.